why_postgres.mdx

```diff
@@ -0,0 +1,124 @@
+
+Secondly, the free tier is important to us because of the company history. The idea of Xata was born out of a [non-profit](https://www.tupu.io/), when we realised that managing databases is a limiting factor for many small projects. Today I'm proud to say that multiple non-profits are building on top of Xata, some on the free tier.
+
+## The platform
+
+It's quite possible to use Xata as a Postgres service and nothing else, and we couldn't be more happy if you do. However, the way we think about Xata is not as a Postgres DBaaS, but as a **data platform for PostgreSQL**: multiple services and tools, integrated together much closer than in a classical cloud provider, simplifying the way developers build data-centric applications and manage databases.
+
+This is the Xata platform at the high level:
+
+![Xata data platform diagram](images/xata-platform.png)
+
+Data is automatically replicated, including schema changes, via logical replication to Elasticsearch, which serves our search, vector search, and aggregations endpoints, so the areas in which it excels. Files are stored in S3 and served through the Cloudflare CDN. The replication events are used to keep a reference count of the files, so they can be shared between branches, yet deleted when they are no longer needed.
+
+If file attachments or full-text search or vector/hybrid search are important for your application, Xata will likely offer you a better platform to build on top than a classical DBaaS. And it's only the beginning, we are planning to have similar tightly-integrated solutions for time-series, intelligent caching, queuing, and more.
+
+## The implementation
+
+One key concept in the Xata implementation is the notion of _logical database branches_. A database can have multiple branches, and always has at least one, called by default `main`. A database branch is represented in Postgres as a [schema](https://www.postgresql.org/docs/current/ddl-schemas.html), and can be allocated on one of many Postgres clusters around the world, as well as moved from one cluster to another.
+
+This results in a level of decoupling between the logical databases and branches as experienced by the application, and the underlying Postgres instances where they are running. Because the clusters are already provisioned, logical databases and branches can be started instantly and allocated and moved depending on the business needs. This unlocks some interesting use-cases, like sharding per tenant where every tenant is a branch, or using a logical database for each micro-service, or automatically creating developer branches for each pull request. Or, and this is important for our free tier, have many small databases and branches share the same relatively large underlying Postgres cluster.
```

Comment:
```suggestion
This results in a level of decoupling between the logical databases and branches as experienced by the application, and the underlying Postgres instances where they are running. Because the clusters are already provisioned, logical databases and branches can be started instantly and allocated and moved depending on the business needs. This unlocks some interesting use cases, like sharding per tenant where every tenant is a branch, using a logical database for each micro-service, or automatically creating developer branches for each pull request. We use this logical separation in our free tier, where many small databases and branches share the same relatively large underlying Postgres cluster.
```